content/webapp/components/WorkDetails/WorkDetails.AvailableOnline.tsx

```diff
@@ -210,30 +258,109 @@ const ItemPageLink = ({
           </Layout>
         )}
       >
-        <ConditionalWrapper
-          condition={isWorkVisibleWithPermission}
-          wrapper={children => (
-            <Space $v={{ size: 'l', properties: ['margin-bottom'] }}>
-              {children}
-            </Space>
+        {!extendedViewer &&
+          ((collectionManifestsCount && collectionManifestsCount > 0) ||
+            (canvasCount && canvasCount > 0)) && (
+            <ConditionalWrapper
+              condition={isWorkVisibleWithPermission}
+              wrapper={children => (
+                <Space $v={{ size: 'l', properties: ['margin-bottom'] }}>
+                  {children}
+                </Space>
+              )}
+            >
+              <p className={font('lr', 6)} style={{ marginBottom: 0 }}>
+                Contains:{' '}
+                {collectionManifestsCount && collectionManifestsCount > 0
+                  ? pluralize(collectionManifestsCount, 'volume')
+                  : canvasCount
+                    ? pluralize(canvasCount, 'image')
+                    : null}
+              </p>
+            </ConditionalWrapper>
           )}
-        >
-          {(Boolean(collectionManifestsCount && collectionManifestsCount > 0) ||
-            Boolean(canvasCount && canvasCount > 0)) && (
-            <p className={`${font('lr', 6)}`} style={{ marginBottom: 0 }}>
-              Contains:{' '}
-              {collectionManifestsCount && collectionManifestsCount > 0
-                ? `${collectionManifestsCount} ${
-                    collectionManifestsCount === 1 ? 'volume' : 'volumes'
-                  }`
-                : canvasCount && canvasCount > 0
-                  ? `${canvasCount} ${canvasCount === 1 ? 'image' : 'images'}`
-                  : ''}
-            </p>
```

Comment:
I just reorganised this section, no changes should occur, but just felt it could be simplified, sorry if it complicates things or breaks anything!